python-avd/pyavd/_eos_designs/shared_utils/cv_topology.py

```diff
@@ -59,51 +60,47 @@ def cv_topology_platform(self: SharedUtilsProtocol) -> str | None:
             return self.cv_topology.platform
         return None
 
+    def get_cv_topology_level(self: SharedUtilsProtocol, node_type: str) -> int:
+        if not (cv_topology_level := self.inputs.cv_topology_levels.get(node_type)):
+            msg = f"'cv_topology_levels' must include all node types when 'use_cv_topology' is set to 'true'. Missing type '{self.type}'."
```

Comment:
We use this function too fetch level for both `self.type` and `neighbor_facts.type`

Removing `spine` from the list results in `Missing type 'l3leaf'`

```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: pyavd._errors.AristaAvdInvalidInputsError: 'cv_topology_levels' must include all node types when 'use_cv_topology' is set to 'true'. Missing type 'l3leaf'.
```

```suggestion
            msg = f"'cv_topology_levels' must include all node types when 'use_cv_topology' is set to 'true'. Missing type '{node_type}'."
```